src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSRUniqueArgumentCollection.java

```diff
@@ -19,6 +23,25 @@
     @Argument(fullName="quantize_quals", shortName = "qq", doc = "Quantize quality scores to a given number of levels", optional=true)
     public int quantizationLevels = 0;
 
+
+    /**
+     * Static quantized quals are entirely separate from the quantize_qual option which uses dynamic binning.
+     * The two types of binning should not be used together.
+     */
+    @Advanced
+    @Argument(fullName="static_quantized_quals", shortName = "SQQ", doc = "Use static quantized quality scores to a given number of levels (with -BQSR)", optional=true)
```

Comment:
Can you make this mutually exclusive with quantize quals if it's bad to use them together?